Packet++/header/DoIpEnums.h

```diff
@@ -0,0 +1,711 @@
+#pragma once
+
+#include <string>
+#include <stdint.h>
+
+/// @file
+
+/**
+ * @namespace pcpp
+ * @brief The main namespace for the PcapPlusPlus lib
+ */
+namespace pcpp
+{
+	/**
+	 * @brief Represents the DoIP (Diagnostics over IP) protocol versions.
+	 */
+	enum class DoIpProtocolVersion : uint8_t
```

Comment:
nit: maybe change the enum values to start with a capital letter? for example: `ReservedVersion` instead of `reservedVersion`